src/index.html

```diff
@@ -6,13 +6,66 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product Cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100..900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__icon"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        src="./images/imac.jpeg"
+      />
+      <div class="product-card__details">
+        <h1 class="product-card__details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="product-card__details-code">Product code: 195434</p>
+      </div>
+      <div class="product-card__reviews">
+        <div class="product-card__reviews-stars product-card__reviews-stars--4">
+          <span class="product-card__reviews-stars-star"></span>
+          <span class="product-card__reviews-stars-star"></span>
+          <span class="product-card__reviews-stars-star"></span>
+          <span class="product-card__reviews-stars-star"></span>
+          <span class="product-card__reviews-stars-star"></span>
+        </div>
+        <p class="product-card__reviews-text">
+          Reviews:
+          <span>5</span>
+        </p>
+      </div>
+      <div class="product-card__price">
+        <span class="product-card__price-label">Price:</span>
+        <span class="product-card__price-value">$2,199</span>
+      </div>
+      <a
+        href="#"
+        class="product-card__buy-button"
+        data-qa="hover"
+      >
+        <span class="product-card__buy-button-text">Buy</span>
```

Comment:
Redundant span wrapper
```suggestion
        Buy
```